app/theme.server.ts

```diff
@@ -2,7 +2,7 @@ import { createCookieSessionStorage } from "remix";
 
 import { Theme, isTheme } from "~/components/ThemeProvider";
 
-const sessionSecret = SESSION_SECRET;
+const sessionSecret = 'SESSION_SECRET';
```

Comment:
Can you set this back to `const sessionSecret = SESSION_SECRET`. That `SESSION_SECRET` global comes from Cloudflare (you can see the type definition in [bindings.t.ds](https://github.com/apihero-run/jsonhero-web/blob/main/app/bindings.d.ts). If you follow the [DEVELOPMENT.md](https://github.com/apihero-run/jsonhero-web/blob/main/DEVELOPMENT.md) guide you should be able to get the app running locally